README.md

```diff
@@ -48,7 +48,7 @@ more. See [the wiki](https://github.com/neovim/neovim/wiki/Installing-Neovim)!
 
 ### Community
```

Comment:
And remove the Reddit link, since it's on the Community page and is not important enough to be at the top of the readme.